demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -2,42 +2,47 @@
 //  flutterPlugin.swift
 //  Runner
 //
-//  Created by Talwinder Kaur on 2022-10-24.
+//  Created by Avast.Inc on 2022-10-24.
 //
 
 import Flutter
 import UIKit
 import Walletsdk
 
-public class SwiftHelloPlugin: NSObject, FlutterPlugin {
+public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
   public static func register(with registrar: FlutterPluginRegistrar) {
-    let channel = FlutterMethodChannel(name: "HelloPlugin", binaryMessenger: registrar.messenger())
-    let instance = SwiftHelloPlugin()
+    let channel = FlutterMethodChannel(name: "WalletSDKPlugin", binaryMessenger: registrar.messenger())
+    let instance = SwiftWalletSDKPlugin()
     registrar.addMethodCallDelegate(instance, channel: channel)
   }
  
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
-      if (call.method == "storeCredentials"){
-          
-          let storageProvider = StorageNewProvider()
-          do {
-              let jsonVC = """
-              {
-                "id": "http://example.edu/credentials/1872"
-              }
-              """.data(using: .utf8)!
-              try storageProvider!.add(jsonVC)
-              print("credential is stored")
-          } catch {
-              result("error while storing verfiable credential")
-          }
-      }
-      
-      if (call.method == "sayHello"){
-          guard let args = call.arguments as? [String : Any] else {return}
-          let name = args["name"] as! String
-          result("Welcome " + name)
+      if (call.method == "createDID"){
+          createDid(result: result)
       }
 
   }
+    public func createDid(result: @escaping FlutterResult) {
+        let didCreator = DidNewCreator()
+        do {
+            let apiCreate = initializeObject(fromType: ApiCreateDIDOpts.self)
+            apiCreate.method = "key"
+            let byteArray: [UInt8] = [19, 14, 81, 157, 226, 93, 94, 94, 61, 151, 246, 226, 20, 86, 41, 139, 164, 203, 99, 96 ,155 ,5, 156, 244, 198, 141, 99, 6, 19, 130, 72, 46]
```

Comment:
Note: this would change based on @DRK3 's latest update. But we can add a follow up PR to update this.